ci/scan-image.yml

```diff
@@ -79,9 +79,11 @@ run:
       instana-twistcli scan ((target-image))
     fi
 
+    export OUTPUT_DIR=$(ls | grep tt-output)
```

Comment:
Is there are chance, that multiple directories would match? That might be problematic for `cat` below.

You could also think about using 
```
export OUTPUT_DIR=$(realpath *tt-output*)
```
if you want to avoid the pipe and use an absolute path for the cat.

Reply:
There will always be only one directory with that name. The pwd is `/tmp/build/<unique-id>`, where unique-id is different for each container/concourse run, so we are fine with using the relative path. BUT, I do like more `export OUTPUT_DIR=$(realpath *tt-output*)`, thanks!